### ISSUE TITLE
508-enhancement [WORKFLOW]: VAOS CC Provider Select - Can we consider making the select a provider routine more explicit?

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#19809]
## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

While testing, I noticed there's a hard requirement to click the "Select provider" button when I make a radio button choice, if I want that provider to persist across screens. I tried clicking a radio button then pressing "Continue" and my provider did not persist. I'm not sure this is a defect, but maybe a user flow to review post-launch.

This feels like a sub-routine that should be explicit and require users to make a choice or exit, like a captive modal or with the button to select the provider visible on the form all the time. Just a suggestion, not a defect.

## Acceptance Criteria

- [ ] Team reviews the issue and opts to revise or close as they feel appropriate.

## Environment

* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/new-appointment/community-care-preferences
* Test user Judy

## Screenshots or Trace Logs

![Screen Shot 2021-02-10 at 4 47 02 PM](https://user-images.githubusercontent.com/934879/107583571-b769f700-6bc0-11eb-8d50-34851a198b09.png)

**Comment by laurenernest** [comment]
Additional design work related to this topic will be handled in #21802 as part of a larger improvement effort.

Closing this ticket as it won't be implemented specifically with this initiative.
